.rubocop_todo.yml

```diff
@@ -0,0 +1,235 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config --no-auto-gen-timestamp`
```

Comment:
Though, this is the command that I likely used to get some idea of what styles were being used...